From 4a84b37ce73a840856aa0df86177f857892a50e6 Mon Sep 17 00:00:00 2001
From: Claude <claude.baek@sk.com>
Date: Fri, 24 Jul 2020 10:34:35 +0900
Subject: [PATCH] remove

---
 TestApp/ViewController.swift | 1 -
 1 file changed, 1 deletion(-)

diff --git a/TestApp/ViewController.swift b/TestApp/ViewController.swift
index 7f2578b..d40ea9c 100644
--- a/TestApp/ViewController.swift
+++ b/TestApp/ViewController.swift
@@ -81,7 +81,6 @@ final class ViewController: UIViewController {
         // addSubview 로 레퍼런스 카운트가 증가하여 인스턴스를 유지한다.
         print(weakView)
         weakView?.removeFromSuperview()
-        print(weakView)
     }
    
 }
-- 
2.23.0

